src/prevector.h

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2015-2019 The Bitcoin Core developers
+// Copyright (c) 2015-2018 The Bitcoin Core developers
```

Comment:
Unless it's a straight cherry pick from another project (showing their dev commit], add the veil copyright to any module touched.

Reply:
yea its straight from bitcoin, so ... does it really matter?